src/test/java/net/openhft/chronicle/wire/utils/dto/AbstractEvent.java

```diff
@@ -5,13 +5,13 @@
 package net.openhft.chronicle.wire.utils.dto;
 
 import net.openhft.chronicle.wire.*;
-import net.openhft.chronicle.wire.converter.Base85;
 import net.openhft.chronicle.wire.converter.NanoTime;
+import net.openhft.chronicle.wire.converter.ShortText;
 
 public class AbstractEvent<E extends AbstractEvent<E>> extends SelfDescribingMarshallable {
-    @Base85
+    @ShortText
```

Comment:
Does it make sense to replace `@Base85` with `@ShortText` on any fields except `String` ?
We're not deprecating `@Base85`, right?

Reply:
We aren't deprecating Base85 yet, however we might in the future.